plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2331,6 +2331,7 @@ elementgui.armor.boots_name=In-game boots name:
 elementgui.armor.layer_texture=<html>Armor layer texture:<br><small>If the list is empty, you need to import or create an armor texture first
 elementgui.armor.equip_sound=Armor equip sound:
 elementgui.armor.max_damage_absorption=Maximal damage that armor absorbs / durability: 
+elementgui.armor.model_translucency=Model supports translucent rendering?
```

Comment:
I would add a warning to only check this parameter if needed as it has performance implications. If there is room, as 2nd line with small html tag as in other parts of the UI